binding.cc

```diff
@@ -151,6 +154,78 @@ int worker_load(worker* w, char* name_s, char* source_s) {
   return 0;
 }
 
+
+MaybeLocal<Module> ResolveCallback(Local<Context> context, Local<String> name, Local<Module> referrer) {
```

Comment:
I'm getting a warning here 
```
binding.cc:158:20: warning: 'ResolveCallback' has C-linkage specified, but returns incomplete type 'MaybeLocal<v8::Module>' which could be incompatible with C [-Wreturn-type-c-linkage]
```
It is fixed if this function is moved outside the `extern "C"`

Reply:
Awesome. Will do.